tests/cupyx_tests/scipy_tests/stats_tests/test_stats.py

```diff
@@ -60,3 +64,173 @@ def test_propotion_too_big(self):
             a = xp.array([4, 8, 2, 0, 9, 5, 10, 1, 7, 3, 6])
             with pytest.raises(ValueError):
                 scp.stats.trim_mean(a, 0.6)
+
+
+@testing.with_requires('scipy>=1.7')
```

Comment:
Please see https://github.com/cupy/cupy/pull/6855/files#r935552655.
```suggestion
@testing.with_requires('scipy')
```